### ISSUE TITLE
Error Upon Deploying to Payara(Glassfish Fork) Server 

### ISSUE THREAD

**Issue by Lanzjess** [primefaces/showcase#50]
I tried to deploy in Payara(Glassfish Fork) Server and I encountered errors:

Exception while loading the app : 
`CDI definition failure:Exception List with 1 exceptions: Exception 0 : javax.enterprise.event.ObserverException at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) at java.lang.reflect.Constructor.newInstance(Constructor.java:423) at java.lang.Class.newInstance(Class.java:442) at org.jboss.weld.security.NewInstanceAction.run(NewInstanceAction.java:33) at java.security.AccessController.doPrivileged(Native Method) at org.jboss.weld.injection.Exceptions.rethrowException(Exceptions.java:40) at org.jboss.weld.injection.Exceptions.rethrowException(Exceptions.java:78) at org.jboss.weld.injection.StaticMethodInjectionPoint.invoke(StaticMethodInjectionPoint.java:96) at org.jboss.weld.injection.Method`

As a work-around, I tried the following:
1. Include in the pom dependencies the following: 

		<dependency>
		    <groupId>org.glassfish.main.web</groupId>
		    <artifactId>weld-integration</artifactId>
		    <version>4.1.2</version>
		</dependency>
        
		<dependency>
		    <groupId>org.glassfish.main.grizzly</groupId>
		    <artifactId>nucleus-grizzly-all</artifactId>
		    <version>4.1.2</version>
		</dependency>
        
		<dependency>
		    <groupId>org.jglue.cdi-unit</groupId>
		    <artifactId>cdi-unit</artifactId>
		    <version>3.1.4</version>
		    <scope>test</scope>
		</dependency>
         <dependency>
        <groupId>org.glassfish.jersey.media</groupId>
        <artifactId>jersey-media-json-jackson</artifactId>
        <version>2.13</version>
        <scope>provided</scope>
    </dependency>

    <dependency>
        <groupId>org.glassfish.jersey.core</groupId>
        <artifactId>jersey-server</artifactId>
        <version>2.13</version>
        <scope>provided</scope>
    </dependency>
2. created beans.xml inside META-INF

**Comment by tandraschko** [comment]
The showcase is designed to run on jetty currently. last month we also included weld in the showcase now.
We have no plans to provide profiles for javaee and plain servlet containers now.
Probably you have the remove the JavaEE dependencies + web.xml listeners.

**Comment by Lanzjess** [comment]
OK thanks tandraschko! I removed the JavaEE dependencies + web.xml listeners but still same error comes out. Anyway, it's fine right now. It's just my test for my real problem where in Ajax functionalities are not working when I deployed in Payara Server as I posted in Stackoverflow https://stackoverflow.com/questions/47157817/upon-converting-netbeans-dynamic-web-project-to-eclipse-maven-project-the-prime .
The pop-up dialog, column filters and menu items in menu bottons are failing.
